cmd/cmd/remoteLogin.go

```diff
@@ -98,5 +106,5 @@ func executeLoginCmd(args []string) {
 
 func init() {
 	remoteCmd.AddCommand(loginCmd)
-	loginCmd.SetHelpTemplate(loginCmdShortDesc + loginCmdExamples)
+	loginCmd.SetHelpTemplate(remoteLoginCmdHelpString)
```

Comment:
Shall we use [1] instead of using remoteLoginUsageString?
[1] https://github.com/wso2/product-mi-tooling/blob/master/cmd/utils/utils.go#L311